cms/djangoapps/contentstore/utils.py

```diff
@@ -156,19 +157,36 @@ def get_lms_link_for_certificate_web_view(course_key, mode):
     )
 
 
+def get_course_authoring_url(course_module):
+    """
+    Gets course authoring microfrontend URL
+    """
+    return configuration_helpers.get_value_for_org(
+        course_module.location.org,
+        'COURSE_AUTHORING_MICROFRONTEND_URL',
+        settings.COURSE_AUTHORING_MICROFRONTEND_URL
+    )
+
+
+def get_pages_and_resources_url(course_module):
+    """
+    Gets course authoring microfrontend URL for Pages and Resources view.
+    """
+    if ENABLE_PAGES_AND_RESOURCES_MICROFRONTEND.is_enabled(course_module.id):
+        mfe_base_url = get_course_authoring_url(course_module)
+        if mfe_base_url:
+            return f'{mfe_base_url}/course/{course_module.id}/pages-and-resources'
+    return None
+
+
 def get_proctored_exam_settings_url(course_module):
     """
     Gets course authoring microfrontend URL for links to proctored exam settings page
     """
-    course_authoring_microfrontend_url = ''
-
     if settings.FEATURES.get('ENABLE_EXAM_SETTINGS_HTML_VIEW'):
```

Comment:
One implication of this change is that now the call to `get_course_authoring_url` does _not_ check if the above flag.
That may be what we want, but I just want to check..

Reply:
I split it out to explicitly avoid this flag. This flag only seems to be relevant for the proctored exams view, so `get_course_authoring_url` is a way to get that URL irrespective of that setting.